platform/install/quick-start-guide.mdx

```diff
@@ -2,67 +2,109 @@
+
+```bash title="Get the current Kubernetes context."
+kubectl config current-context
+```
 
-```bash
+Once you've confirmed the correct context, deploy the platform by running:
+
+```bash title="Deploy the platform."
 vcluster platform start
 ```
-This will deploy vCluster Platform into the host `vcluster-platform` namespace.
+
+This will deploy the platform onto the host-cluster in the `vcluster-platform` namespace.
 The UI will automatically open in your browser and log you in. You will be asked for your user details to create the admin user.
 
+:::info
+The deployment process typically takes about 5-10 minutes, depending on your cluster's resources and network speed.
+:::
+
+You should see output similar to the following:
+
+```bash title="Platform installation output."
+...
+
+##########################   LOGIN   ############################
+
+Username: admin
+Password: 27177595-21bc-4ff9-9f3a-51e0f722408b  # Change via UI or via: vcluster platform reset password
+
+# highlight-start
+Login via UI:  https://hth45c8.loft.host
+Login via CLI: vcluster login https://hth45c8.loft.host
+# highlight-end
+
+#################################################################
+
+vCluster Platform was successfully installed and can now be reached at: https://hth45c8.loft.host
+Thanks for using vCluster Platform!
+11:38:33 done You are successfully logged into vCluster Platform!
+- Use `vcluster create` to create a new virtual cluster
+- Use `vcluster platform add vcluster` to add an existing virtual cluster to a vCluster platform instance
+```
+
+This command deploys the platform into the `vcluster-platform` namespace of your Kubernetes cluster. Upon successful deployment, the UI will automatically open in your default web browser. You will be prompted to create an admin user by providing your user details.
+
 :::tip Troubleshooting
-Your vCluster Platform pod is not starting or the UI is not showing up? View the
-[Troubleshooting Guide](../use-platform/troubleshooting) if you have any
-issues.
+If your platform pod is not starting or the UI is not showing up, view the [Troubleshooting Guide](../use-platform/troubleshooting) for assistance.
 :::
 
-## 3. Logging in
+## Logging in
 
-While `vcluster platform start` logs you in automatically, you can also manually login via the CLI:
+While `vcluster platform start` logs you in automatically, you can also manually log in via the CLI:
 
-```bash
+```bash title="Platform login"
 vcluster login https://1rtjxak.loft.host      # See `vcluster platform start` output for login credentials and your actual sub-domain name (auto-generated).
 ```
 
 This command will open the browser where you can sign in using the login data shown in the
 output of `vcluster platform start`.
 
+## Create Virtual Clusters
```

Comment:
Can we ensure we add this into all of the install sections? 

Currently missing in Helm docs.